bundles/org.openhab.binding.powermax/src/main/resources/OH-INF/thing/zone.xml

```diff
@@ -13,6 +13,7 @@
 
 		<label>Alarm Zone</label>
 		<description>This thing represents a physical device such as a door, window or a motion sensor.</description>
+		<semantic-equipment-tag>AlarmDevice</semantic-equipment-tag>
```

Comment:
It is not an alarm device, it is an alarm zone is there no tag appropriate for that ?

Reply:
Sorry, this one is good.